test/framework/integration_framework/fake_peer/network/yac_network_notifier.hpp

```diff
@@ -9,14 +9,7 @@
 #include <rxcpp/rx.hpp>
 
 #include "consensus/yac/transport/yac_network_interface.hpp"
-
-namespace iroha {
-  namespace consensus {
-    namespace yac {
-      struct VoteMessage;
-    }  // namespace yac
-  }    // namespace consensus
-}  // namespace iroha
+#include "framework/integration_framework/fake_peer/types.hpp"
```

Comment:
Please move that header to .cpp and do a forward declaration of the type here.

Reply:
The file `fake_peer/types.hpp` consists of type aliases and forward declarations. It is designed to be included in header files of fake peer components.